dosenet.sh

```diff
@@ -23,13 +23,17 @@ PUBLICKEY=$CONFIGDIR/id_rsa_lbl.pub
 case "$1" in
   start)
     logger --stderr --id --tag $LOGTAG "Starting DoseNet script"
+    ntp-wait -n 3 -s 30
```

Comment:
Since it seems to take longer sometimes to update NTP, let's make this `-n 10`. So, up to 5 minutes.